java/debugger.jpda.truffle/nbproject/project.properties

```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 javac.compilerargs=-Xlint:unchecked
-javac.source=1.8
+javac.release=11
```

Comment:
someone would have to verify that bumping the debugger module is ok. But since there is no comment above the property it must be ok.

Reply:
I read a tad of irony into this. However there is an artifact `JPDATruffleBackend.jar` build as part of the build process for that module. For that compilation the the source and target values are hardcoded to 1.8:

https://github.com/apache/netbeans/blob/a5fa02d5e704f77ad32bb0c85815120474331e3b/java/debugger.jpda.truffle/build.xml#L42-L47

The resulting jar indeed only holds class files with version 52 (JDK 8).

From that perspective this looks sane to me.